pygmt/src/blockm.py

```diff
@@ -111,6 +112,14 @@ def blockmean(data=None, x=None, y=None, z=None, outfile=None, **kwargs):
 
     {I}
 
+    summary: str
+        [**m**\|\ **n**\|\ **s**\|\ **w**].
+        Type of summary values calculated by blockmean.
+        - **n**: report the number of input points inside each block
+        - **s**: report the sum of all z-values inside a block
+        - **w**: report the sum of weights
+        - **m**: report mean value [Default]
```

Comment:
Now it looks like this:

![image](https://user-images.githubusercontent.com/3974108/140678398-8a202beb-fe4a-4727-b2e6-960d7d5e3726.png)

I believe a new line is still needed at the start of the bullet.